### PR TITLE
Disable nav tree activation on the element select

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
#1238 

[percy](https://percy.io/8fe2b2f1/share/builds/35432989/)

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

**Comment by OlegMoshkovich** [comment]
..In the process of running Percy

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

**Comment by OlegMoshkovich** [comment]
This change is testable in Percy; however, it is difficult to test since it involves synchronization of the model interaction and the tree panel response. The current test involves accessing the app through the permalink and SCREEN testing whether all of the elements respond accordingly. The current PR changes the interaction with the tree, keeping it closed when an element is selected; therefore, a Percy screenshot has been adjusted.

**Unit Testing:**
Here, I don't know exactly which component to test because this PR subtracts the logic that sets the panel visibility store value to true, keeping it the same. Maybe that could be tested by setting up the store variables between selected elements and the tree and testing if those store variables remain unchanged when setting the selected elements to a number.

**End-to-End Test:**
Here, to test the interaction, an element needs to be clicked. We need to figure out how to select an element. One way is to navigate to the app with the element selected in the URL and test if the tree remains closed. Yes, that test could be adjusted.

image

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

File: cypress/e2e/view-100/synchronized-view-and-navtree.cy.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ describe('View 100: Synchronized View and NavTree', () => {
       waitForModelReady('twoLevelSelect')
     })
 
-    it('Item highlighted in tree and scene - Screen', () => {
+    it('Item highlighted in scene - Screen', () => {
       cy.percySnapshot()
```
Comment (pablo-mayrgundter) [inline cypress/e2e/view-100/synchronized-view-and-navtree.cy.js:50]: Please add check that tree is not visible

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]

File: cypress/e2e/view-100/synchronized-view-and-navtree.cy.js (hunk 1 above)
Comment (OlegMoshkovich) [reply cypress/e2e/view-100/synchronized-view-and-navtree.cy.js:51]: done.

**Review by OlegMoshkovich — COMMENTED** [review]
Added.

**Review by pablo-mayrgundter — APPROVED** [review]

File: cypress/e2e/view-100/synchronized-view-and-navtree.cy.js (hunk 1 above)
Comment (OlegMoshkovich) [reply cypress/e2e/view-100/synchronized-view-and-navtree.cy.js:51]: done

**Pull request merged**